functionary/ui/urls.py

```diff
@@ -64,15 +64,20 @@
         (tasks.TaskDetailView.as_view()),
         name="task-detail",
     ),
+    path(
+        "task/<uuid:pk>/results",
+        (tasks.TaskResultsView.as_view()),
+        name="task-results",
+    ),
     path("team_list/", (teams.TeamListView.as_view()), name="team-list"),
     path(
         "team/<uuid:pk>",
         (teams.TeamDetailView.as_view()),
         name="team-detail",
     ),
-    path(
-        "environment/set_environment_id",
-        (environments.set_environment_id),
+    re_path(
+        r"^environment/set_environment/(?:(?P<pk>\w+)?$)",
```

Comment:
The environment id seems to actually get sent in the body, not the url, so it seems like this could just be a regular `path` similar to what it was before.  Am I missing something?